ReadMe.md

```diff
@@ -106,6 +106,14 @@ To select | 選択（する） | せんたく |
 To search | 検索（する） | けんさく |
 Object Oriented Programming (OOP) | オブジェクト指向 | オブジェクトしこう |
 Functional Programming (FP) | 関数型プログラミング | かんすうがたプログラミング |
+Inherit (from X) | （Xを）継承（する） | けいしょう |
+Type | 型 | かた | as in data types
+Static | 静的 | せいてき |
+Dynamic | 動的 | どうてき |
```

Comment:
Are these "static" and "dynamic" in the broadest sense of the word? Or do they have a very specific usage domain?